main.py

```diff
@@ -243,6 +308,7 @@ def run(play):
     play = run(1)
 
 # finish up and save
+
```

Comment:
```suggestion
```

I don't really want this space between the comment and doing stuff.

Reply:
Sure